server/python/app.py

```diff
@@ -53,7 +53,8 @@ def get_config():
         'stripePublishableKey': os.getenv('STRIPE_PUBLISHABLE_KEY'),
         'stripeCountry': os.getenv('STRIPE_ACCOUNT_COUNTRY') or 'US',
         'country': 'US',
-        'currency': 'eur'
+        'currency': 'eur',
+        'paymentMethods': os.getenv('PAYMENT_METHODS').split(', ') or ['card']
```

Comment:
This throws if you don't have PAYMENT_METHODS in the .env (like if you're me who ran this a long time ago and am using my old .env 🙃). Can we make this a ternary: 

```
os.getenv('PAYMENT_METHODS').split(', ') if os.getenv('PAYMENT_METHODS') else ['card']
```

Reply:
Fixed!